recipes/litehtml/all/conanfile.py

```diff
@@ -146,7 +146,7 @@ def package_info(self):
             self.cpp_info.components["litehtml_litehtml"].requires.append("icu::icu")
 
         if True: # FIXME: remove once we use a vendored gumbo library
-            self.cpp_info.components["gumbo"].set_property("cmake_target_name", "gumbo")
+            self.cpp_info.components["gumbo"].set_property("cmake_target_name", "litehtml::gumbo")
```

Comment:
same comment, tartget is not namespaced.

Reply:
Thanks for pointing this out, I overlooked the definitions in the build module